src/graph/DeleteVerticesExecutor.cpp

```diff
@@ -93,11 +109,12 @@ void DeleteVertexExecutor::deleteEdges(std::vector<storage::cpp2::EdgeKey>* edge
     std::move(future).via(runner).thenValue(cb).thenError(error);
 }
 
-void DeleteVertexExecutor::deleteVertex() {
-    auto future = ectx()->getStorageClient()->deleteVertex(spaceId_, vid_);
+void DeleteVerticesExecutor::deleteVertices() {
+    auto future = ectx()->getStorageClient()->deleteVertices(spaceId_, vids_);
```

Comment:
the parameter of deleteVertices is passed by value, so here maybe std::move(vids_) is better?

Reply:
thanks & fixed